tests/sbc_test.py

```diff
@@ -76,6 +76,80 @@ def simulator(theta):
     get_nltp(thetas, xs, posterior)
 
 
+@pytest.mark.slow
+@pytest.mark.parametrize("prior", ("boxuniform", "independent"))
+@pytest.mark.parametrize("method, sampler", [(SNLE, "vi")])
```

Comment:
```suggestion
@pytest.mark.parametrize("method", (SNPE,))
```

Reply:
I would use SNPE to make things fast. we only want to test SBC here.

Reply:
I tried that at first too, but with SNPE the number of samples and posterior runs had to be increased 5-10 fold. Even though the test is marked as slow already, I decided that this is too much of an effort.

Reply:
But SNLE with VI requires retraining VI for every new `x`. NPE is amortized. 
Changing to `SNPE`, even with 1000 simulations and 1000 posterior samples reduces the test run time from 8s (SNLE) to 3,4s (SNPE) on my machine.

Reply:
Ok, I'll check.

Reply:
OK, SNPE runs 13s on my laptop. I guess, you have the better hardware. Fixes are incoming.

Reply:
addressed with 89e1ffd1